### PR TITLE
Add Satellite component and update the version of backend and ui

### PR DESCRIPTION
**Pull request by mrproliu**

### CODE REVIEW

File: deploy/platform/docker/Makefile [hunk 1]
```diff
@@ -22,6 +22,11 @@ include Makefile.in
 features := $(subst $(comma), ,$(FEATURE_FLAGS))
 features := $(foreach f,$(features),-f docker-compose.$(f).yaml)
 
+BACKEND_ADDRESS := oap:11800
```
Comment (kezhenxu94) [inline deploy/platform/docker/Makefile:25]: 1. `BACKEND_ADDRESS` contains the port

File: deploy/platform/kubernetes/Makefile [hunk 2]
```diff
@@ -22,6 +22,11 @@ include Makefile.in
 features := $(subst $(comma), ,$(FEATURE_FLAGS))
 features := $(foreach f,$(features),feature-$(f))
 
+BACKEND_SERVICE := oap
+ifneq (,$(findstring satellite,$(features)))
+	BACKEND_SERVICE := satellite
+endif
```
Comment (kezhenxu94) [inline deploy/platform/kubernetes/Makefile:28]: 2. `BACKEND_ADDRESS` doesn't contain the port, please make (1) and (2) consistent

File: deploy/platform/kubernetes/Makefile.in [hunk 3]
```diff
@@ -21,4 +21,4 @@
 NAMESPACE ?= default
 AGENTLESS ?= false
 
-FEATURE_FLAGS ?= java-agent-injector,cluster,kubernetes-monitor,so11y,vm,als,event,istiod-monitor
\ No newline at end of file
+FEATURE_FLAGS ?= java-agent-injector,cluster,kubernetes-monitor,so11y,vm,als,event,istiod-monitor,satellite
```
Comment (kezhenxu94) [inline deploy/platform/kubernetes/Makefile.in:24]: Please disable satellite by default

File: deploy/platform/docker/Makefile.in [hunk 4]
```diff
@@ -17,4 +17,4 @@
 #
 
 .EXPORT_ALL_VARIABLES:
-FEATURE_FLAGS ?= agent,cluster,so11y,vm,event
+FEATURE_FLAGS ?= agent,cluster,so11y,vm,event,satellite
```
Comment (kezhenxu94) [inline deploy/platform/docker/Makefile.in:20]: Please disable satellite by default

File: deploy/platform/kubernetes/Makefile.in (hunk 3 above)
Comment (wu-sheng) [reply deploy/platform/kubernetes/Makefile.in:24]: Do we have a proper way to set agent backend address when satellite is or is not disable

**Comment by wu-sheng** [comment]
@kezhenxu94 Please add `satellite` as default activated in the real demo deployment.

**Review by kezhenxu94 — APPROVED** [review]

**Comment by kezhenxu94** [comment]
> @kezhenxu94 Please add `satellite` as default activated in the real demo deployment.

If there is new visualization of Satellite in the new UI I can add it, but does Satellite adds anything new in current UI?

**Comment by wu-sheng** [comment]
Yes, we have a menu and dashboard for Satallite self observability, which is why I request this.

**Comment by kezhenxu94** [comment]
> Yes, we have a menu and dashboard for Satallite self observability, which is why I request this.

Got it

**Comment by wu-sheng** [comment]
Check https://github.com/apache/skywalking/pull/8776, this is what we expect.

**Pull request merged**